### ISSUE TITLE
Convert game "web site" into external links

### ISSUE THREAD

**Issue by dfabulich** [iftechfoundation/ifdb-suggestion-tracker#469]
Today, we have a "web site" field for all games, and we have a separate box for "external links." But the web site is just another external link, which should appear in the box.

* We should probably move the "Web site" link out of the header at the top of the page and display it as the top link in the page
* Then, we should run a big SQL script to convert "Web site" links to External links, making them the first link in games that have them (but deduplicating them if the web site already appears as an external link)
* Then, we can edit pages to demote the web site below the downloads, if necessary

**Comment by dougo** [comment]
I agree that the way the web site link is displayed now is not great, but I think it's a useful field to denote the official, canonical site for a game, rather than just lumping it in with the other external links.

**Comment by dfabulich** [comment]
One thing we could do is give the official web site some special visual treatment in the External Links list, so it wouldn't feel "lumped in." Perhaps a highlighted color, or slightly larger font size; something above and beyond being first on the list.

… but I still think we'd want/need more configurability than that.

For many games, the "official, canonical site" is less useful than the IFDB entry; it's nothing more than an Inform-generated "game with website," containing the game's description and a link to download. (That's information we already displayed on IFDB.) In cases like that, I think the "Web Site" link should be less important than a download link to the IF Archive; the Play Online button should link to Parchment.

For some games, the web-based Vorple version is the primary and most important one, so _that_ should appear at the top, and the Play Online button should link to that.

So, it seems plausible that we might agree that:

* The official web site is an External Link, and it belongs in the External Links box
* The official web site shouldn't feel "lumped in" with less-important External Links. It should have a special visual treatment, and, by default, it should be the topmost External Link.
* The official web site may or may not be the most important link in External Links. Users should be able to edit the page to demote the official web site below other, more important/useful links.

If we _did_ agree to all that (though, we might not agree to it yet?), I think the way to handle this would be for the "Add a Link" dialog to have a new feature: there could be a checkable box for "this is an official link." Official links would get the "special visual treatment" I described above.

If you like the sound of that, we could do it in phases:

1. Move the Web Site to the External Links box with a special visual treatment, pinned to the top.
    At this point, there would be two bugs:

    * The Web Site might be duplicated in External Links (once as the "Official Web Site" and then again as an ordinary "External Link")
    * The official web site would always be pinned to the top, even if it isn't the best/most important link to highlight
2. Update "Add a Link" to support "official links."
3. Mass-convert each "Web Site" link to an External Links with "official link" checked, deduplicating any existing External Link.

How does that sound?

**Comment by jtn20** [comment]
This sounds like it'd be possible to have a single link, marked as both "official" and "playable game", for the common case of itch.io pages for web-playable games.

Good; it's bothered me for a while entering that twice, in "Web site" (source of latest version, updates, news etc) and "External links" (so that we have a "Play on-line" button pointing to the official latest version).
